src/main/java/edu/ucsb/cs156/gauchoride/entities/User.java

```diff
@@ -31,4 +31,8 @@ public class User {
   private boolean admin;
   @Builder.Default
   private boolean driver=false;
+
+  public void setDriver(boolean driver) {
+    this.driver = driver;
+}
```

Comment:
Indentation problem still not fixed.  Should be lined up with `public` on line 35.

```suggestion
  }
```

Reply:
I apologize, I thought it was the spacing between the boolean on line 34 so I thought I resolved it. I will fix asap.

Reply:
Now, it should be resolved.